src/attributes.js

```diff
@@ -124,7 +125,7 @@ var defaults = {
 var mutators = {
   // Special generic mutator that's called for any attribute that does not
   // have a specific mutator.
-  __all: applyAttributeTyped,
+  [symbols.all]: applyAttributeTyped,
```

Comment:
You'll need this on [Line 102](`https://github.com/sparhami/incremental-dom/blob/5cffa31d80bae76a66589471cb6334906940379c/src/attributes.js#L102`), too.